openpype/settings/defaults/project_anatomy/attributes.json

```diff
@@ -9,18 +9,6 @@
     "resolutionWidth": 1920,
     "resolutionHeight": 1080,
     "pixelAspect": 1.0,
-    "applications": [
-        "maya_2020",
-        "nuke_12-2",
-        "nukex_12-2",
-        "hiero_12-2",
-        "resolve_16",
-        "houdini_18-5",
-        "blender_2-90",
-        "harmony_20",
-        "photoshop_2021",
-        "aftereffects_2021",
-        "unreal_4-24"
-    ],
+    "applications": [],
```

Comment:
This is not a good idea. All it will cause is that everyone will be confused on the first run. On the contrary the default should have latest version of each application

Reply:
Done